requirements.txt

```diff
@@ -44,6 +44,9 @@ pytz==2017.2
 pyyaml==3.12              # via watchdog
 requests-toolbelt==0.8.0  # via twine
 requests==2.18.4          # via requests-toolbelt, sphinx, twine
+ruamel.ordereddict==0.4.13 ; python_version < "3.0" # via ruamel.yaml
+ruamel.yaml==0.15.37
+simplejson==3.14.0
```

Comment:
Should `simplejson` be part of our `requirements.in`?

Reply:
It's actually a transitive dependency from brewtils. Looks like `pip-tools` doesn't correctly add the 'via' comment when the primary dependency is installed from github.